### ISSUE TITLE
When I use Extension weppy-BS3, I got an Error like this

### ISSUE THREAD

**Issue by laozhu** [gi0baro/weppy#113]
When I comment `app.use_extension(BS3)` line, everything is OK and I got correct config json result.

```python
from weppy import App
from weppy.tools import service
from weppy_haml import Haml
from weppy_bs3 import BS3
from weppy_assets import Assets

app = App(__name__)

app.config.Haml.set_as_default = True
app.config.Haml.auto_reload = True
app.config.Assets.out_folder = "dist"
# app.use_extension(BS3)
app.use_extension(Assets)

@app.route('config')
@service.json
def config():
    return app.config
```

When I uncommnet this line, I got an Error below, I didn't do anything else, maybe a extension bug?

![0576ca1f-08e3-4b14-903f-7c18cc9ebe4a](https://cloud.githubusercontent.com/assets/1212457/14175756/cea339f0-f77f-11e5-964f-fd8e37e0e434.png)

**Comment by gi0baro** [comment]
Hi @laozhu, seems you found a bug in weppy. Will merge #114 (thanks @cassiobotaro) and make a bugfix release later today.

**Comment by gi0baro** [comment]
@laozhu you can find the fix on current master and 0.6.3 release

**Comment by laozhu** [comment]
@gi0baro I have pip upgrade weppy to latest version, Thank you :+1:
